docs/annotations.md

```diff
@@ -73,3 +77,90 @@ spec:
           serviceName: websocket-repeater
           servicePort: 80
 ```
+
+# Connection Draining
+`connection-draining`: This annotations allows to specify whether to enable connecitn draining and the timeout after which
+`connection-draining-timeout`: This annotations allows to specify a timeout after which Application Gateway will terminate the requests to the draining backend endpoint.
```

Comment:
s/annotations/annotation

This annotation allows a user to specify a timewout after which the Application Gateway will terminate the request to the draining backend.